vault/expiration.go

```diff
@@ -1017,8 +1017,7 @@ func (m *ExpirationManager) revokeEntry(le *leaseEntry) error {
 	}
 
 	// Handle standard revocation via backends
-	resp, err := m.router.Route(logical.RevokeRequest(
-		le.Path, le.Secret, le.Data))
+	resp, err := m.router.Route(logical.RevokeRequest(le.Path, le.Secret, le.Data))
```

Comment:
This comes as a non-sequitur :-D

Reply:
Heh, yeah at one point there was a context being passed here, but pulled it out for a different PR